packages/protocol/tsconfig.json

```diff
@@ -13,9 +13,10 @@
     "removeComments": false,
     "skipLibCheck": true,
     "sourceMap": true,
-    "target": "es5",
-    "downlevelIteration": true
+    "target": "ES2020",
```

Comment:
could this by why things are not working?

Reply:
maybe, let me roll it back, pretty sure I changed it for a reason though